imap_processing/ialirt/tests/unit/test_decom_ialirt.py

```diff
@@ -0,0 +1,91 @@
+import sys
+from pathlib import Path
+
+import pytest
+
+from imap_processing.ialirt.l0.decom_ialirt import decom_packets, generate_xarray
+
+
+@pytest.fixture(scope="session")
+def packet_path():
+    """Returns the raw packet directory."""
+    return Path(sys.modules[__name__.split(
+        '.')[0]].__file__).parent / 'ialirt' / 'tests' / 'test_data' / 'l0' / \
+        'IALiRT Raw Packet Telemetry.txt'
```

Comment:
Do we need to look at the module name, or can we just look at the file path itself? Also below.
```suggestion
    return Path(__file__).parent.parent / 'test_data' / 'l0' / \
        'IALiRT Raw Packet Telemetry.txt'
```
Or you could use the variable that comes with the package: https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/65050d2f5905d78708f2deafa73c9d7e9f2ff970/imap_processing/__init__.py#L10